config/theme.yml

```diff
@@ -67,16 +67,19 @@ global_settings:
         - ps_searchbar
       displayHome:
         - ps_imageslider
+        - ps_customtext
         - ps_featuredproducts
         - ps_banner
-        - ps_customtext
+        - ps_newproducts
+        - ps_bestsellers
       displayFooterBefore:
         - ps_emailsubscription
         - ps_socialfollow
       displayFooter:
         - ps_linklist
         - ps_customeraccountlinks
         - ps_contactinfo
+        - ~
```

Comment:
Oh, what does this means?

Reply:
Keep whatever is currently hooked to it.
In this very case, first ps_linklist, then ps_customeraccountlinks, then ps_contactinfo, then whatever is currently hooked to it

Reply:
https://devdocs.prestashop.com/1.7/themes/getting-started/theme-yml/